### DISCUSSION TITLE
Data migrations & upgrades

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#14744] [General]
I'm wondering how people are handling data upgrades or migrations. My app is in Alpha stage, and if the data structure changes enough I can just write a quick LinqPad app to upgrade things, but I'm trying to think of strategies to automate this when we're in production.

Just curious to see what people here do for this kind of action.

**Comment by ayende** [comment]
You may want to look here: https://github.com/migrating-ravens/RavenMigrations

**Reply by ItsRobbAllen** [reply]
Thanks, that looks pretty much like what I need!
